js-exercises/diff-array/diffArray.js

```diff
@@ -0,0 +1,23 @@
+function diffArray(firstArray, secondArray) {
+  const firstUnique = firstArray.reduce((acc, el) => {
+    if (secondArray.indexOf(el) === -1) {
+      return [...acc, el];
```

Comment:
You could have just pushed to the existing array instead of creating a new array everytime